### ISSUE TITLE
Automapping and worlds

### ISSUE THREAD

**Issue by basxto** [bjorn/tiled#2869]
It would be nice if automapping could replace tiles on map edges.
This especially interesting for drawing paths with automapping, since they usually connect over maps.

**Comment by bjorn** [comment]
Right, at the moment there's a few options about how edges should be dealt with, but it can't inspect neighboring maps.

Since rules sometimes depend on each other as well, I guess to fully support worlds one would need to apply the automapping rules to multiple maps at once as well. :-S

I think in general, you'll want to set `OverflowBorder`, which should work most of the time, assuming the same tiles are present on the connecting map.
